### PR TITLE
Smoke tests

### PR DESCRIPTION
**Pull request by danworth**
### What problem does this pull request solve?
Creates a basic smoke test of runner by filling in a simple form and checking that it submits.

Trello card: https://trello.com/c/DtJRJvlt/1414-smoke-test-for-forms-runner

### Things to consider when reviewing
You can run this new test from your machine with the following.
```
LOG_LEVEL=DEBUG SMOKE_TEST_FORM_URL='https://submit.forms.service.gov.uk/form/2570/scheduled-smoke-test' GUI=1 bundle exec rspec spec/features/smoke_test_runner_spec.rb
```

This seems to have been running fine in dev for the last few weeks. You can see it in the Codebuild or the CloudWatch alarms named `scheduled_smoke_tests_failing` and `scheduled_smoke_tests_running`

We could extend it in future to cover more question types, but the e2e tests should be aimed at the more comprehensive testing of different question types. The smoke tests are better off being as simple as possible to test the critical journeys we want to cover.

Note that the parameters were moved in parameter store so this updates the tests to use the new names: https://github.com/alphagov/forms-deploy/pull/604

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Has all relevant documentation been updated?

### CODE REVIEW

File: spec/features/smoke_test_runner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    expect(question_to_be_answered?).to be true
+
+    question_number = 1
+    while question_to_be_answered?
+      logger.info "Answering question: #{question_number}"
+      question_type = page.first('input')[:name]
+      fill_in(class: 'govuk-input', with: answer_for(question_type))
+      click_button 'Continue'
+      expect(page).to have_no_content('There is a problem')
```
Comment (danworth) [inline spec/features/smoke_test_runner_spec.rb:24]: Curious on opinions on this please, might not be bullet-proof enough, e.g if we change the content etc...

Comment (danworth) [reply spec/features/smoke_test_runner_spec.rb:27]: I've decided to check that after clicking 'continue' the page is not the same as it was before, e.g. it hasn't reloaded with some kind of validation error. If at any point we redirect to the Technical Errors page then that page will not have a question on it, and it won't have the check your answers text so the test will fail.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Vim related files
+tags
+*.swp
```
Comment (lfdebrux) [inline .gitignore:3]: It feels like these ignores are not necessarily related to this repo, have you considered putting them in a global gitignore file on your machine instead?

File: bin/load_env_vars.sh [hunk 3]
```diff
@@ -47,7 +47,8 @@ function set_env_vars() {
 
   export FORMS_ADMIN_URL="$(admin_url $environment)"
   export PRODUCT_PAGES_URL="$(product_pages_url $environment)"
-  export SETTINGS__GOVUK_NOTIFY__API_KEY="$(get_param /${environment}/smoketests/notify/api-key)"
-  export AUTH0_EMAIL_USERNAME="$(get_param /${environment}/smoketests/auth0/email-username)"
-  export AUTH0_USER_PASSWORD="$(get_param /${environment}/smoketests/auth0/auth0-user-password)"
+  export SETTINGS__GOVUK_NOTIFY__API_KEY="$(get_param /${environment}/automated-tests/e2e/notify/api-key)"
+  export AUTH0_EMAIL_USERNAME="$(get_param /${environment}/automated-tests/e2e/auth0/email-username)"
+  export AUTH0_USER_PASSWORD="$(get_param /${environment}/automated-tests/e2e/auth0/auth0-user-password)"
```
Comment (lfdebrux) [inline bin/load_env_vars.sh:52]: Can we split this change out into a separate commit?

File: spec/support/feature_helpers.rb [hunk 4]
```diff
@@ -5,6 +5,27 @@
...
+    when 'question[number]'
+      '123'
+    when 'question[email]'
+      'smoke_test@example.com'
+    when 'question[full_name]'
+      'smoke_test'
+    when 'question[text]'
+      'smoke_test'
+    else
+      raise "Unsuppored question type: #{question_type}. Only number, email,
```
Comment (DavidBiddle) [inline spec/support/feature_helpers.rb:23]: Typo:
```suggestion
      raise "Unsupported question type: #{question_type}. Only number, email,
```

Comment (danworth) [reply spec/support/feature_helpers.rb]: Thanks David! 👍

**Review by DavidBiddle — APPROVED** [review]
Code looks good and test runs correctly on my machine 💯
